File: sop-sdk/sdk-csharp/SDKCSharp/Common/CustomDataNameBuilder.cs
using System;
namespace SDKCSharp.Common
{
    /// <summary>
    /// 返回固定的dataName
    /// </summary>
    public class CustomDataNameBuilder: DataNameBuilder
    {

        private string dataName = "result";

        public CustomDataNameBuilder()
        {
        }


        public CustomDataNameBuilder(string dataName)
        {
            this.dataName = dataName;
        }

        public string Build(string method)
        {
            return dataName;
        }
    }
}

File: sop-sdk/sdk-java/src/main/java/com/gitee/sop/sdk/common/CustomDataNameBuilder.java
package com.gitee.sop.sdk.common;

/**
 * 返回固定的
 * {
 *     "result": {
 *         "code": "20000",
 *         "msg": "Service Currently Unavailable",
 *         "sub_code": "isp.unknown-error",
 *         "sub_msg": "系统繁忙"
 *     },
 *     "sign": "ERITJKEIJKJHKKKKKKKHJEREEEEEEEEEEE"
 * }
 * @author tanghc
 */
public class CustomDataNameBuilder implements DataNameBuilder {
    private String dataName = "result";

    public CustomDataNameBuilder() {
    }

    public CustomDataNameBuilder(String dataName) {
        this.dataName = dataName;
    }

    @Override
    public String build(String method) {
        return dataName;
    }
}
